.werft/observability/monitoring-satellite.ts

```diff
@@ -22,7 +22,7 @@ const sliceName = 'observability';
 /**
  * installMonitoringSatellite installs monitoring-satellite, while updating its dependencies to the latest commit in the branch it is running.
  */
-export async function installMonitoringSatellite(params: InstallMonitoringSatelliteParams) {
+export function installMonitoringSatellite(params: InstallMonitoringSatelliteParams) {
```

Comment:
All those changes to `async` seem to be unrelatable with this PR, could we leave that to when we tackle https://github.com/gitpod-io/gitpod/issues/9059?

Reply:
I reverted all changes that I made to the async functions.